source/install/troubleshooting.rst

```diff
@@ -35,6 +66,7 @@ Put together a timeline to eliminate events prior to the error/problem occurring
     - Is the problem occurring only for a user who was recently added to the environment, such as a new employee?
     - Do differences exist between the users who are affected and the users who are not affected?
 
+You can also search the error messages online - existing solutions can often be applied.
```

Comment:
```suggestion
You can also search the error messages online - existing solutions can often be applied.
```
Wondering if we could rather link to the forum which may very well have the solution?

Reply:
This is taken from the original Troubleshooting section. I'm 0/5 on this.

Reply:
I think we need to be a bit more explicit here as well.  "You can also search the error messages online.  Existing solutions from our forum <link> and other forum sites can often be found and applied."  0/5 on my suggestion, but at least we are pointing the reader towards more specified resources then "online".

Reply:
It sais `If, after following these guidelines, you're still facing the issue, visit the `Troubleshooting Forum` two sentences later, would that not be double information? 

I'd remove the following two sentences and only leave the one suggested above, but include the link. WDYT?

Reply:
One suggestion is to make the changes and we can see they fit in contextually? Easier to see it written out and go from there.

Reply:
That should already be the case, no?

> You can also search the error messages online. Existing solutions from our `forum <https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150>`_ and other forum sites can often be found and applied.

Reply:
Ah awesome, I didn't see that. :) Do we have a list of other forum sites that we'd recommend? If not, perhaps we can remove it?

Reply:
afaik there is no such list. I'm fine removing it.

Reply:
Thanks @faase! It's looking good. :)